crypto/trace.c

```diff
@@ -227,8 +228,8 @@ static int set_trace_data(int category, int type, BIO **channel,
     char *curr_prefix = NULL;
     char *curr_suffix = NULL;
 
-    /* Ensure ossl_trace_init() is called */
-    OPENSSL_init_crypto(0, NULL);
+    /* Ensure do_ossl_trace_init() is called once */
+    ossl_trace_init();
```

Comment:
Just make this a call to `RUN_ONCE(&trace_inited, ossl_init_trace);` directly, and we can remove `ossl_trace_init` entirely.

Reply:
Done.